DataFormats/TrackReco/interface/TrackExtraBase.h

```diff
@@ -73,12 +78,15 @@ class TrackExtraBase
 
     }
 
+    TrajParams const & trajParams() const  {return m_trajParams;}
+    Chi2sFive const & chi2s() const { return m_chi2s;}

```

Comment:
I'm not so sure about char used here 
https://its.cern.ch/jira/browse/CMSTRACK-150?focusedCommentId=1311785&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-1311785